scripts/monitor_ci.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+/**
+ * Part of E2E Monitor test run for staging and master builds
+ * Script queries the repository to discover whether a contract
+ * published to Goerli in CI has been picked up and saved by the
+ * monitor.
+ */
+
+const assert = require('assert');
+const fetch = require('node-fetch');
+const util = require('util');
+const log = console.log;
+
+// TODO: Check if multiple repositories and toggle by staging or master branch
+const root = 'https://contractrepostaging.komputing.org/';
```

Comment:
There is a FQDN in config and we could also add repository endpoint.

For example: https://github.com/ethereum/source-verify/blob/be2a631b8200df28ec96d778d3e2f63529d5a596/environments/.env.latest#L2

Reply:
@edisinovcic Ah cool - I saw `REPOSITORY_URL` in the envs and it's "staging" for both latest and stable. Is it safe to make those different? If so I can add that here.

https://github.com/ethereum/source-verify/blob/cccf260c229cee2b4850ece83734e3be7b5197b1/environments/.env.stable#L25

Reply:
Definitely, latest is for staging and stable is for master so please update it.